cli/packages/cmd/export.go

```diff
@@ -93,10 +93,33 @@ var exportCmd = &cobra.Command{
 			IncludeImport: includeImports,
 		}
 
+		
+		var secrets []models.SingleEnvironmentVariable
+		var isUserSession bool
+		var infisicalDotJson models.WorkspaceConfigFile
+		var userBackupSecretsEncryptionKey []byte
+		var loggedInUserDetails util.LoggedInUserDetails
+
 		if token != nil && token.Type == util.SERVICE_TOKEN_IDENTIFIER {
 			request.InfisicalToken = token.Token
 		} else if token != nil && token.Type == util.UNIVERSAL_AUTH_TOKEN_IDENTIFIER {
 			request.UniversalAuthAccessToken = token.Token
+		} else {
+			isUserSession = true
+			loggedInUserDetails, err = util.GetCurrentLoggedInUserDetails()
+			if err != nil {
+				util.HandleError(err)
+			}
+
+			projectConfig, err := util.GetWorkSpaceFromFile()
+			if err != nil {
+				util.HandleError(err)
+			}
+
+			infisicalDotJson = projectConfig
+
+			userBackupSecretsEncryptionKey = []byte(loggedInUserDetails.UserCredentials.PrivateKey)[0:32]
+			secrets = util.GetBackupSecretsIfDisconnected(infisicalDotJson.WorkspaceId, environmentName, userBackupSecretsEncryptionKey)
```

Comment:
How come we didn't just put this function inside of `GetAllEnvironmentVariables()`? Kind of like we did before. I think we cut could down on complexity if we were to keep this logic inside the `GetAllEnvironmentVariables()` method.